PrimerIPay88SDK/lib/MH/IpayPayment.h

```diff
@@ -13,7 +13,7 @@
     NSString *prodDesc;
     NSString *userName;
     NSString *userEmail;
-    NSString *userContact;
+//    NSString *userContact;
```

Comment:
Same as above

Reply:
This is how the new iPay88 framework is deployed, this is one of the changes in their header files. I naturally commented out all the usages in our files.